be/src/olap/rowset/segment_v2/page_handle.h

```diff
@@ -46,42 +46,49 @@ class PageHandle {
             : _is_data_owner(false), _cache_data(std::move(cache_data)) {}
 
     // Move constructor
-    PageHandle(PageHandle&& other) noexcept
-            : _is_data_owner(false),
-              _data(std::move(other._data)),
-              _cache_data(std::move(other._cache_data)) {
+    PageHandle(PageHandle&& other) noexcept : _cache_data(std::move(other._cache_data)) {
         // we can use std::exchange if we switch c++14 on
```

Comment:
_is_data_owner is not inited, do not know it is true or false, so what is swap means?

Reply:
`_is_data_owner` is initialized to false at definition time before the body of the copy constructor is executed

swap has no special meaning, it has the same meaning as a = b, b = false